curriculum/challenges/english/14-responsive-web-design-22/learn-the-css-box-model-by-building-a-rothko-painting/step-042.md

```diff
@@ -7,6 +7,7 @@ dashedName: step-42
 
 # --description--
 
+Use `border-radius` on `.two` to set its top-left and bottom-right radius to `8px`, and top-right and bottom-left radius to `10px`.
 Set the `border-radius` of `.two` to `8px 10px`. This will round its top-left and bottom-right corners by 8 pixels, and top-right and bottom-left corners by 10 pixels.
```

Comment:
First sentence, or the whole line might be obsolete now.

Reply:
Apologies. I forgot to remove the original line.

Reply:
No worries :+1: